LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022 Team DashX (https://dashx.com/)
```

Comment:
Is this MIT license

Reply:
Yes. Snapped from Alamofire (MIT license) & verified with the standard license.